iset.mm

```diff
@@ -117423,8 +117486,619 @@ Infinite sums (cont.)
       SIFYGYROUGABWSSYBVSVTYMYRYPSMRZAYQSYPMWBQYMYSASMRAYMYPASMAWCWDAWETWFTWHVB
       TWNYMXGXDOYOYMXGXCGKLZMRZXDYMGIFZXGUUAOWGABWSGYBVSVTYTGXCMWIQWJWHVBTWNABW
       SGYBWNWKUUBWNWGVDWLWM $.
+
+    $d A n $.
+    $( The exponential function of a positive real number is greater than 1
+       plus that number.  (Contributed by Mario Carneiro, 14-Mar-2014.)
+       (Revised by Mario Carneiro, 30-Apr-2014.) $)
+    efgt1p $p |- ( A e. RR+ -> ( 1 + A ) < ( exp ` A ) ) $=
+      ( vn vk vy wcel c1 caddc cn0 cv cexp co cfa cfv cdiv cc0 cc wceq eftvalcn
+      1e0p1 eqtrd crp cmpt cseq ce clt rpcn fveq2i cuz 0nn0 eleqtri a1i elnn0uz
+      nn0uz wa eqid eftcl eqeltrd sylan2br addcl adantl seq3p1 0zd seq3-1 mpan2
+      eft0val 1nn0 fac1 oveq2i exp1 oveq1d div1 syl5eq syl5eqr oveq12d effsumlt
+      syl id eqbrtrrd ) AUAEZFGBHABIZJKVTLMNKUBZOUCZMZFAGKZAUDMUEVSAPEZWCWDQAUF
+      WEWCOFGKZWBMZWDFWFWBSUGWEWGOWBMZWFWAMZGKWDWECDGPWAOOOOUHMZEWEOHWJUIUMUJUK
+      CIZWJEWEWKHEZWKWAMZPEWKULWEWLUNWMAWKJKWKLMNKPABWAWKWAUOZRAWKUPUQURZWKPEDI
+      ZPEUNWKWPGKPEWEWKWPUSUTZVAWEWHFWIAGWEWHOWAMZFWECDGPWAOWEVBWOWQVCWEWRAOJKO
+      LMNKZFWEOHEWRWSQUIABWAOWNRVDAVETTWEWIFWAMZAFWFWASUGWEWTAFJKZFLMZNKZAWEFHE
+      ZWTXCQVFABWAFWNRVDWEXCXAFNKZAXBFXANVGVHWEXEAFNKAWEXAAFNAVIVJAVKTVLTVMVNTV
+      LVPVSABWAFWNVSVQXDVSVFUKVOVR $.
+
+    $( The exponential function of a positive real number is greater than 1.
+       (Contributed by Paul Chapman, 21-Aug-2007.)  (Revised by Mario Carneiro,
+       30-Apr-2014.) $)
+    efgt1 $p |- ( A e. RR+ -> 1 < ( exp ` A ) ) $=
+      ( crp wcel c1 caddc co ce cfv 1red cr 1re readdcl sylancr reefcld clt wbr
+      rpre ltaddrp mpan efgt1p lttrd ) ABCZDDAEFZAGHUBIUBDJCZAJCUCJCKAQZDALMUBA
+      UENUDUBDUCOPKDARSATUA $.
   $}
 
+  ${
+    $d x y A $.  $d x y B $.
+    $( The exponential function on the reals is strictly monotonic.
+       (Contributed by Paul Chapman, 21-Aug-2007.)  (Revised by Jim Kingdon,
+       20-Dec-2022.) $)
+    efltim $p |- ( ( A e. RR /\ B e. RR ) ->
+                 ( A < B -> ( exp ` A ) < ( exp ` B ) ) ) $=
+      ( cr wcel clt wbr ce cfv w3a cmin co cmul c1 crp simp2 cc0 syl reefcld cc
+      recnd simp1 resubcld biimp3a elrpd efgt1 wb efgt0 ltmulgt11 syl3anc mpbid
+      posdif caddc wceq efadd syl2anc pncan3d fveq2d eqtr3d breqtrd 3expia ) AC
+      DZBCDZABEFZAGHZBGHZEFVAVBVCIZVDVDBAJKZGHZLKZVEEVFMVHEFZVDVIEFZVFVGNDVJVFV
+      GVFBAVAVBVCOZVAVBVCUAZUBZVAVBVCPVGEFABUKUCUDVGUEQVFVDCDVHCDPVDEFZVJVKUFVF
+      AVMRVFVGVNRVFVAVOVMAUGQVDVHUHUIUJVFAVGULKZGHZVIVEVFASDVGSDVQVIUMVFAVMTZVF
+      VGVNTAVGUNUOVFVPBGVFABVRVFBVLTUPUQURUSUT $.
+
+    $( The exponential function on the reals is strictly monotonic.
```

Comment:
is increasing
(or, depending on (i)set.mm's conventions, "nondecreasing")

Reply:
Changed in both set.mm and iset.mm.